src/data-card-advanced.css.ts

```diff
@@ -0,0 +1,147 @@
+import {style} from '@vanilla-extract/css';
+import {sprinkles} from './sprinkles.css';
+import * as mq from './media-queries.css';
+import {vars} from './skins/skin-contract.css';
+import {applyAlpha} from './utils/color';
+
+export const anchorCard = style([
+    {
+        selectors: {
+            '&:focus:after': {
+                outline: '2px solid black',
+            },
+            '&:after': {
+                content: ' ',
+                position: 'absolute',
+                top: '0',
+                left: '0',
+                width: '100%',
+                height: '100%',
+                cursor: 'pointer',
+                zIndex: '1',
+            },
+        },
+    },
+]);
+
+export const actions = style([
+    sprinkles({
+        display: 'flex',
+        flex: 1,
+        justifyContent: 'space-between',
+    }),
+    {flexWrap: 'wrap'},
+]);
+
+export const boxed = style([
+    sprinkles({display: 'flex'}),
+    {
+        isolation: 'isolate', // Needed to preserve border-radius with Video component and Safari
+        selectors: {
+            '&:hover': {
+                backgroundColor: vars.colors.backgroundContainerHover,
```

Comment:
You are applying hover without onpress state, that's a bit weird
https://tinyurl.com/29g89hfx

Use hover only when the card has onPress interaction